package.json

```diff
@@ -208,49 +204,33 @@
     "jest-environment-jsdom": "^28.1.3",
     "lint-staged": "^15.2.1",
     "moment-timezone": "^0.5.43",
-    "postcss": "^8.1.10",
-    "postcss-loader": "^4.1.0",
     "prettier": "^2.2.0",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
     "react-test-renderer": "^18.1.0",
-    "rollup": "^2.79.1",
-    "rollup-plugin-copy": "^3.4.0",
-    "rollup-plugin-insert": "^1.3.1",
-    "rollup-plugin-node-globals": "^1.4.0",
-    "rollup-plugin-peer-deps-external": "^2.2.4",
-    "rollup-plugin-terser": "^7.0.2",
-    "rollup-plugin-url": "^3.0.1",
-    "rollup-plugin-visualizer": "^4.2.0",
     "semantic-release": "^19.0.5",
     "stream-chat": "^8.21.0",
     "style-loader": "^2.0.0",
     "ts-jest": "^28.0.8",
-    "typescript": "^4.7.4",
-    "url-loader": "^4.1.1",
-    "webpack": "4.44.2",
-    "webpack-cli": "^3.3.12",
-    "webpack-dev-server": "~3.11.0"
+    "typescript": "^5.4.2",
+    "url-loader": "^4.1.1"
```

Comment:
`url-loader` can go away as well :)

Reply:
I think every `*-loader` package should be removed, as they are typically used with weback.

Reply:
All gone!